src/relay/pass/unifier.h

```diff
@@ -0,0 +1,142 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file include/tvm/relay/pass/unifier.h
+ * \brief The type unifier which solves a system of equations between
+ * incomplete types.
```

Comment:
unifier is no longer needed as the logic is now in solver

Reply:
rebase issue. removed.